data/rbac-schema/src/main/java/net/corda/rbac/schema/RbacKeys.java

```diff
@@ -56,4 +64,47 @@ private RbacKeys() {
      * FQN for a flow to be started.
      */
     public static final String FLOW_NAME_REGEX = "[._$a-zA-Z0-9]{1,250}";
+
+    /**
+     * Flow state regex.
+     */
+    public static final String FLOW_STATE_REGEX = "[_a-zA-Z0-9]{3,255}";
+
+    /**
+     * Certificate usage regex.
+     */
+    public static final String CERTIFICATE_USAGE_REGEX = "(p2p-tls|p2p-session|code-signer)";
+
+    /**
+     * Tenant ID regex.
+     */
+    public static final String TENANT_ID_REGEX = "(" + UUID_CHARS + "{12}|p2p)";
+
+    /**
+     * Key ID regex.
+     */
+    public static final String KEY_ID_REGEX = UUID_CHARS + "{12}";
+
+    /**
+     * HSM category regex.
+     */
+    public static final String HSM_CATEGORY_REGEX = "(ACCOUNTS|CI|LEDGER|NOTARY|PRE_AUTH|SESSION_INIT|TLS|JWT_KEY)";
```

Comment:
I wonder if we would be better off with `[A-Z_]{2,20}` to allow for future extension of this set?